### DISCUSSION TITLE
Limit size of default completer

### DISCUSSION THREAD

**Discussion by rhz** [nushell/nushell#14167] [Q&A]
Hi. Sometimes the default completer takes up the whole screen. I'd like to limit the maximum size it can have. Is there a way to do this?

**Comment by ghost** [comment]
you should be able to adjust the [default completion menu config](https://www.nushell.sh/book/line_editor.html#completion-menu)  
copying the [layout from history](https://www.nushell.sh/book/line_editor.html#history-menu) would for example limit it to 10

**Reply by rhz** [reply]
That didn't work. First, I like the columnar layout for the default completion menu. I tried adding `page_size: 10` to that layout and it didn't do anything. Then I did as you suggested and replaced the columnar layout by the list layout. The effect was completely unexpected to me. Instead of showing the possible completions, now tab shows the history. That's very strange.
